hack/validation/labels.sh

```diff
@@ -29,6 +29,13 @@ rule=${rule//\"/\\\"}            # escape double quotes
 rule=${rule//$'\n'/}             # remove newlines
 rule=$(echo "$rule" | tr -s ' ') # remove extra spaces
 
+# nodepool
+# v1beta1
+printf -v expr '.spec.versions[1].schema.openAPIV3Schema.properties.spec.properties.template.properties.metadata.properties.labels.x-kubernetes-validations +=
```

Comment:
Should we have antoher check that checks that `.spec.versions` is length 2?

So if there's another new version (or we remove a version?) we get a failure?

Reply:
Added